extensions/amp-analytics/0.1/vendors.js

```diff
@@ -71,20 +71,52 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'afsanalytics': {
+    'vars': {
+      'server': 'www',
+      'websiteid': 'xxxxxxxx',
+      'event': 'click',
+      'clicklabel': 'clicked from AMP page',
+    },
+    'transport': {'beacon': false, 'xhrpost': false, 'image': true},
+    'requests': {
+      'host': '//${server}.afsanalytics.com',
+      'base': '${host}/cgi_bin/',
+      'pageview': '${base}connect.cgi?usr=${websiteid}Pauto' +
+      '&js=1' +
```

Comment:
please add two spaces to lines that are continuations of previous ones. That way, it is easier to read exactly where a request starts and ends.